frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@bcgov/bc-sans": "1.0.1",
+    "@mdi/react": "^1.5.0",
```

Comment:
same comment as earlier, please confirm that this is necessary. We do not want to increase our bundle size unless absolutely necessary.